### PR TITLE
adwcleaner: Add version 8.2

### PR DESCRIPTION
**Pull request by cristianrz**
Added AdwCleaner 8.2:
* Not sure about the license, should I just add "Freeware"?
* Couldn't find a way to download a specific version, rather the link will always download the latest version.

### CODE REVIEW

File: bucket/adwcleaner.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": "8.2",
+    "description": "Free Adware Cleaner & Removal tool",
+    "homepage": "https://www.malwarebytes.com/adwcleaner",
+    "license": {
+        "identifier": "Unknown"
```
Comment (issaclin32) [inline bucket/adwcleaner.json:6]:
```suggestion
        "identifier": "Freeware",
        "url": "https://www.malwarebytes.com/eula"
```

File: bucket/adwcleaner.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        ]
+    ],
+    "checkver": {
+        "url": "https://support.malwarebytes.com/hc/en-us/sections/360005837474-Release-History-News",
+        "regex": "Malwarebytes AdwCleaner (\\d[.\\d]+)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://download.toolslib.net/download/direct/1/latest?channel=release"
```
Comment (issaclin32) [inline bucket/adwcleaner.json:28]:
```suggestion
                "url": "https://download.toolslib.net/download/direct/1/latest?channel=release#/adwcleaner.exe"
```

**Comment by issaclin32** [comment]
> * Not sure about the license, should I just add "Freeware"?

Exactly. If the software is free to use, it is in the category of "Freeware"

> * Couldn't find a way to download a specific version, rather the link will always download the latest version.

This would not be a problem. Many packages are tracked in this way.

**Comment by cristianrz** [comment]
Both changes comitted. Thanks.

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**